packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -954,6 +954,26 @@ abstract class CustomClipper<T> {
   String toString() => '$runtimeType';
 }
 
+/// A [CustomClipper] that clips to the outer path of a [ShapeBorder].
+class ShapeBorderClipper extends CustomClipper<Path> {
+  /// Creates a [ShapeBorder] clipper.
+  ///
+  /// The [shapeBorder] argument must not be null.
+  const ShapeBorderClipper({
+    @required this.shapeBorder
+  }) : assert(shapeBorder != null);
+
+  final ShapeBorder shapeBorder;
```

Comment:
please run `flutter analyze --dartdocs` to catch any places where you are adding undocumented members

Reply:
done